server/@types/viewModels/index.d.ts

```diff
@@ -47,6 +47,7 @@ declare module 'viewModels' {
   export interface FunctionalSkills {
     problemRetrievingData: boolean
     assessments: Array<Assessment>
+    prisonNumber: string
```

Comment:
Add prisonNumber to the FunctionalSkills view object - because we are now holding it in the session we need to know which prisoner it is for